writing-apps/the-basic-setup.md

```diff
@@ -90,10 +90,23 @@ sudo apt install elementary-sdk
 
 On elementary OS 6 beta and newer, you should already have the required Flatpak remote and platform pre-installed. On earlier versions or other OSes, you can add the remote and install the Flatpak platform and SDK:
 
-```bash
-flatpak remote-add --if-not-exists --system appcenter https://flatpak.elementary.io/repo.flatpakrepo
-flatpak install -y appcenter io.elementary.Platform io.elementary.Sdk
-```
+{% tabs %}
+    {% tab title="OS 8 and Above" %}
+
+    {% endtab %}
```

Comment:
Sdk has to be installed manually even on OS 8. Only the platform is pre-installed

Reply:
SDK install instructions have been added now